types/hubot/index.d.ts

```diff
@@ -217,20 +246,23 @@ export class Robot<A extends Adapter = Adapter> {
     listen(matcher: (message: Message) => boolean, callback: ListenerCallback<A>): void;
     listen(matcher: (message: Message) => boolean, options: {}, callback: ListenerCallback<A>): void;
     listenerMiddleware(middleware: MiddlewareHandler<A>): void;
+    load(path: string): Promise<void>;
+    loadAdapter(): Promise<void>;
+    loadAdapter(adapterPath: string): Promise<void>;
     loadExternalScripts(packages: string[]): void;
-    loadFile(directory: string, fileName: string): void;
-    loadHubotScripts(path: string, scripts: string[]): void;
-    messageRoom(room: string, ...strings: string[]): void;
+    loadFile(directory: string, fileName: string): Promise<void>;
+    messageRoom(room: string, ...strings: string[]): Promise<void> | Promise<unknown>;
```

Comment:
`Promise<void> | Promise<unknown>` is a strange type; I would think `Promise<unknown>` would suffice?

Reply:
I'll make it a `Promise<any>`. The code of Hubot states it returns what the adapter returns:

https://github.com/hubotio/hubot/blob/main/src/Robot.mjs#L630-L633